assets/src/edit-story/components/colorPicker/colorPicker.js

```diff
@@ -73,7 +73,14 @@ const Body = styled.div`
   padding: ${CONTAINER_PADDING}px;
 `;
 
-function ColorPicker({ color, hasGradient, hasOpacity, onChange, onClose }) {
+function ColorPicker({
+  color,
+  hasGradient,
+  hasOpacity,
+  onChange,
+  onClose,
+  addActions,
```

Comment:
Could we rename this to `renderFooter` or something like that? Then the name doesn't really express what invokers should do with it - it's just something to be rendered in the footer.